cts/scheduler/xml/696.xml

```diff
@@ -108,7 +108,7 @@
         <lrm_resources id="lrm_resources.id22329">
           <lrm_resource id="child_DoFencing:0" class="stonith" type="ssh">
             <lrm_rsc_op id="DoFencing-child_DoFencing-0_start_0" op-digest="e6d0cb6a5fe061b101eb79be4a67facf" operation="start" interval="0" transition-key="0:1:0:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx" crm_feature_set="2.0" transition-magic="0:0;0:1:0:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx" call-id="4" rc-code="0" op-status="0" on_node="hadev2"/>
-            <lrm_rsc_op id="DoFencing-child_DoFencing-0_monitor_5000" op-digest="4346e25e4ed927b0a3ac932538c1dae2" interval="5000" operation="monitor" op-status="-1" rc-code="-1" transition-key="-1:1:0:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx" crm_feature_set="2.0" transition-magic="0:0;-1:1:0:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx" call-id="-1" on_node="hadev2"/>
+            <lrm_rsc_op id="DoFencing-child_DoFencing-0_monitor_5000" op-digest="4346e25e4ed927b0a3ac932538c1dae2" interval="5000" operation="monitor" op-status="-1" rc-code="1" transition-key="-1:1:0:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx" crm_feature_set="2.0" transition-magic="0:0;-1:1:0:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx" call-id="-1" on_node="hadev2"/>
```

Comment:
Yeah, I fixed it where it was initially matching but didn't bother with the ones that already didn't match. At some point it would be nice to check the consistency of all the tests in regard to transition-key/transition-magic. Though I really dislike those fields and would eventually like to drop them (some of the individual components are already separate attributes, and the rest should be).

Reply:
probably wouldn't be that hard to do the consistency checking and updating with a Python regex script. It is a nonzero amount of time though